src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -109,9 +109,12 @@ void setAccessibilityIncrements(List<String> accessibilityIncrements) {
   @Override
   public void onPopulateAccessibilityEvent(AccessibilityEvent event) {
     super.onPopulateAccessibilityEvent(event);
-    if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED ||
-        (event.getEventType() == AccessibilityEvent.TYPE_VIEW_SELECTED && this.isAccessibilityFocused())) {
-      this.setupAccessibility((int) mValue);
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+      if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED ||
+          (event.getEventType() == AccessibilityEvent.TYPE_VIEW_SELECTED && this.isAccessibilityFocused())) {
+          this.setupAccessibility((int)mValue);
```

Comment:
I allowed myself to rebase your changes @MarkCSmith and I adjusted them to the latest of `setupAccessibility` method.